tests/pytorch_tests/model_tests/feature_models/layer_fusing_test.py

```diff
@@ -0,0 +1,247 @@
+# Copyright 2022 Sony Semiconductors Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from abc import ABC
+import torch
+import torch.nn as nn
+from torch.nn import Conv2d, ReLU, Tanh, SiLU, Sigmoid, Linear
+from torch.nn.functional import relu, relu6
+import model_compression_toolkit as mct
+from tests.pytorch_tests.model_tests.base_pytorch_feature_test import BasePytorchFeatureNetworkTest
+from model_compression_toolkit.tpc_models.default_tp_model import get_op_quantization_configs
+
+tp = mct.target_platform
+
```

Comment:
We have no test here for LayerFilterParams cases (where a specific layer type should be fused). Please test some of these patterns as well

Reply:
added LayerFilterParams  to tests